### ISSUE TITLE
Modal does not provide backdrop w/ bootstrap 3.3.2

### ISSUE THREAD

**Issue by newmanw** [angular-ui/bootstrap#3378]
Looks like bootstrap 3.3.2 does not have the bottom 0 anymore on modal-backdrop class

```css
.modal-backdrop {
  bottom: 0;
}
```

This means that the modal dialog does not calc the correct height for the backdrop fade.

plunker here:
http://plnkr.co/edit/5vPyWJvl1Xzs4bYAAebC?p=preview

**Comment by bsweeney** [comment]
Dup of #3184? Discussion there indicates that this has been addressed in Bootstrap for a future release. Temporary fix is to override the Bootstrap styling for the backdrop.

```
.modal-backdrop {
    bottom: 0px;
    position: fixed;
}
```

**Comment by wesleycho** [comment]
This appears to be a Bootstrap issue with a simple workaround - feel free to reopen this issue with another comment if you believe a fix should be in UI Bootstrap.

**Comment by antoinepairet** [comment]
As pointed out in:
https://github.com/angular-ui/bootstrap/issues/2830#issuecomment-82261304
A new bootstrap version is out: https://github.com/twbs/bootstrap/releases/tag/v3.3.4

which solves the issue, see https://github.com/twbs/bootstrap/blob/f5beebe726aa8c1810015d8c62931f4559b49664/less/modals.less#L65

**Comment by evilaliv3** [comment]
i've retested this over the GlobaLeaks project and i confirm and the issue is solved in bootstrap 3.3.4
